.github/workflows/main.yml

```diff
@@ -17,21 +17,15 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    env:
-      ARTIFACT_NAME: application-package
-    outputs:
-      artifactName: ${{ env.ARTIFACT_NAME }}
     steps:
       - uses: actions/checkout@v3
-      - name: Setup
-        uses: ./.github/actions/setup
+      - uses: ./.github/actions/setup
         with:
           enable-build-cache: 'true'
           install-jdk: 'true'
-      - name: Build Swagger CodeGen
-        run: yarn build:swagger-gen
-      - name: Build
-        run: yarn build
+      - run: yarn build:swagger-gen
```

Comment:
why did you remove the task names?

Reply:
I removed it because I think that `Build` is not necessarily a better name than the default `Run yarn build`